src/Sulu/Bundle/WebsiteBundle/Analytics/AnalyticsManager.php

```diff
@@ -66,11 +77,14 @@ public function find($id)
 
     public function create($webspaceKey, $data)
     {
-        $entity = new Analytics();
+        /** @var AnalyticsInterface $entity */
+        $entity = $this->analyticsRepository->createNew();
```

Comment:
Missing in the `createNew` in the `AnalyticsRepositoryInterface`.

Reply:
derived from `RepositoryInterface` see https://github.com/sulu/sulu/pull/5968/files#diff-77a097117b1e5256caea7c780d12fea32d121876b6c841c6465226b353577869R16